### PR TITLE
Story-845: Migrate to Maven Central

### PR DESCRIPTION
**Pull request by JayasriR**

### CODE REVIEW

File: codefresh.yml [hunk 1]
```diff
@@ -23,12 +57,12 @@ steps:
   Build:
     title: Maven build
     fail_fast: false
-    image: regnosys-docker-registry.jfrog.io/regnosys/rosetta-products-deps:java-17
+    image: maven:3.8.2-jdk-11
```
Comment (ja6a-regnosys) [inline codefresh.yml:60]: java 11? 17 was the old one

File: pom.xml [hunk 2]
```diff
@@ -1,18 +1,78 @@
...
+        </developer>
+    </developers>
 
     <properties>
+        <gpg.keyname>configured-by-release-profile</gpg.keyname>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.enforced.version>[17,18)</java.enforced.version>
-        <maven.compiler.release>8</maven.compiler.release>
-
+        <maven.compiler.release>11</maven.compiler.release>
```
Comment (ja6a-regnosys) [inline pom.xml:74]: this could be a problem

File: pom.xml [hunk 3]
```diff
@@ -444,6 +521,66 @@
                 </executions>
             </plugin>
         </plugins>
+        <pluginManagement>
+        <plugins>
```
Comment (ja6a-regnosys) [inline pom.xml:525]: poor layout - needs a tab indentation

File: pom.xml [hunk 4]
```diff
@@ -444,6 +521,66 @@
...
+                        <id>attach-javadocs</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+    </plugins>
+        </pluginManagement>
```
Comment (ja6a-regnosys) [inline pom.xml:583]: ditto

**Review by ja6a-regnosys — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (ja6a-regnosys) [reply pom.xml:74]: why not keep it at 8?

**Review by ja6a-regnosys — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (JayasriR) [reply pom.xml:74]: Might has well upgrade to the latest?

**Review by ja6a-regnosys — APPROVED** [review]

**Pull request merged**